startupscript/install-cli.sh

```diff
@@ -33,21 +47,16 @@ readonly TERRA_SERVER
 if ! command -v wb &> /dev/null; then
   emit "Installing the Workbench CLI ..."
 
-  # If the server environment is a verily server, use the verily download script.
-  if [[ "${TERRA_SERVER}" == *"verily"* ]]; then
-    # Map the CLI server to appropriate AFS service path and fetch the CLI distribution path
-    if ! versionJson="$(curl -s "https://${TERRA_SERVER/verily/terra}-axon.api.verily.com/version")"; then
-      >&2 echo "ERROR: Failed to get version file from ${TERRA_SERVER}"
-      exit 1
-    fi
-    cliDistributionPath="$(echo "${versionJson}" | jq -r '.cliDistributionPath')"
-
-    ${RUN_AS_LOGIN_USER} "curl -L https://storage.googleapis.com/${cliDistributionPath#gs://}/download-install.sh | TERRA_CLI_SERVER=${TERRA_SERVER} bash"
-    cp wb "${WORKBENCH_INSTALL_PATH}"
-  else
-    >&2 echo "ERROR: ${TERRA_SERVER} is not a known Workbench server"
+  # Map the CLI server to appropriate AFS service path and fetch the CLI distribution path
+  if ! versionJson="$(curl -s "$(get_axon_version_url "${TERRA_SERVER}")")"; then
```

Comment:
We've had a convention in the scripts of UPPER_CASE variables being for script global and lower_case variables being function local variables. Is that intentionally not being preserved here or could we dpo that?